### PR TITLE
Rac 1984 cit test fix2

### PR DESCRIPTION
**Pull request by lacarb**
Correction for an earlier CIT test modification.
This should make CIT green for both master and this API fix - https://github.com/RackHD/on-http/pull/613

### CODE REVIEW

File: test/tests/api/v2_0/nodes_tests.py [hunk 1]
```diff
@@ -294,6 +294,7 @@ def test_node_workflows_post(self):
         resps = []
         Api().nodes_get_all()
         nodes = self.__get_data()
+        print nodes
```
Comment (VulpesArtificem) [inline test/tests/api/v2_0/nodes_tests.py]: did you want to keep this print statement?

Comment (lacarb) [inline test/tests/api/v2_0/nodes_tests.py]: I'll cut it out

**Comment by hohene** [comment]
We need PR #613 to merge before this, correct?

**Comment by lacarb** [comment]
@VulpesArtificem, print has been removed. I'm not sure what's up with #613

**Comment by lacarb** [comment]
@RackHD/corecommitters

**Pull request closed without merging**